recipes/perl/meta.yaml

```diff
@@ -0,0 +1,25 @@
+package:
+  name: perl
+  version: 5.20.3.1
+
+source:
+  fn:  strawberry-perl-5.20.3.1-32bit-portable.zip [win]
```

Comment:
yes.  That's been a generally safe assumption for me, but it is an assumption that may not be true.  It is true on OS X, and on our docker build image for linux.